NuGet/package/Fitbit.dev.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>Fitbit.NET</id>
-    <version>2.0.1-RC2-DEV-$buildNumber$</version>
+    <version>2.0.0$buildNumber$-RC2-Dev</version>
```

Comment:
This nuspec is only used when packaging dev-experimental builds (a.k.a builds that have not been pulled into master/main branch, or in our current case async-portable). When packaging the stable version we use PROD.nuspec

Sent from Outlook Mobile




On Tue, Feb 23, 2016 at 12:11 AM -0800, "Adam Storr" <notifications@github.com> wrote:












In NuGet/package/Fitbit.dev.nuspec:
> @@ -2,7 +2,7 @@
>  <package >
>    <metadata>
>      <id>Fitbit.NET</id>
> -    <version>2.0.1-RC2-DEV-$buildNumber$</version>
> +    <version>2.0.0$buildNumber$-RC2-Dev</version>



RC3? Why needDev in the version?



—
Reply to this email directly or view it on GitHub.